### PR TITLE
Fixed input recorder crash

### PR DESCRIPTION
**Pull request by druiz17**
Fixed input recorder crash

### CODE REVIEW

**Comment by druiz17** [comment]
Test plan
- run inputRecorder.js
- click the IRecord button from the button
- click the load button...find and select the file below
- click play recording 
- watch the recording and if the recording repeats without crashing: QA Passed

[ajt-cabbage-patch-chest.json.gz](https://github.com/highfidelity/hifi/files/997628/ajt-cabbage-patch-chest.json.gz)

File: libraries/controllers/src/controllers/InputRecorder.cpp [hunk 1]
```diff
@@ -25,7 +25,7 @@
 
 QString SAVE_DIRECTORY = QStandardPaths::writableLocation(QStandardPaths::AppLocalDataLocation) + "/" + BuildInfo::MODIFIED_ORGANIZATION + "/" + BuildInfo::INTERFACE_NAME + "/hifi-input-recordings/";
 QString FILE_PREFIX_NAME = "input-recording-";
-QString COMPRESS_EXTENSION = ".tar.gz";
```
Comment (hyperlogic) [inline libraries/controllers/src/controllers/InputRecorder.cpp]: The extension should be .json.gz

**Comment by hifi-gustavo** [comment]
Build console output for commit: 9a0fd78e41bc80fccb19a36c605d729f0b707be1 available at: https://gist.github.com/4040d98463632dcfae76bff4977afff0<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10452-9a0fd78.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10452-9a0fd78.dmg

**Comment by YuppyPlays** [comment]
Testing.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 1f328cc923d756aa9d2085da64f375433127330a available at: https://gist.github.com/4040d98463632dcfae76bff4977afff0<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10452-1f328cc.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10452-1f328cc.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 6155d31513e2764182cfb854a0db85f4630b777d available at: https://gist.github.com/4040d98463632dcfae76bff4977afff0<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10452-6155d31.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10452-6155d31.dmg

**Comment by YuppyPlays** [comment]
Testing complete:

The InputRecording.js loaded up and then the .gz file given runs the recorded actions great and replayed without crashing. **Pass**

However, in-case its worth noting, if the window is clicked off of, rather than just having a lower fps, you can see the motions being played are physically slowed down, not sure if this is meant to happen?

Also, the game crashed the first 2 times opening up with PR build and entering sandbox, this was literally within 5 seconds of booting the game up, 3rd time round though it stopped crashing, error reports sent with info to accompany the crashes.

Let me know if there is anything else I can do/provide :)

**Pull request merged**